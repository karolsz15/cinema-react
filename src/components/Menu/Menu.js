import React from 'react';
import { NavLink } from 'react-router-dom';

function Menu() {
  return (
    <div className="menu">
      <ul>
        <li>
          <NavLink to="/" exact>
            <div className="home">
              <i className="home" />
            </div>
          </NavLink>
        </li>
        <li>
          <NavLink to="/trailers">
            <div className="video">
              <i className="videos" />
              <i className="videos1" />
            </div>
          </NavLink>
        </li>
        <li>
          <NavLink to="/booking">
            <div className="bk">
              <i className="booking" />
              <i className="booking1" />
            </div>
          </NavLink>
        </li>
        <li>
          <NavLink to="/contact">
            <div className="cnt">
              <i className="contact" />
              <i className="contact1" />
            </div>
          </NavLink>
        </li>
      </ul>
    </div>
  );
}

export default Menu;
