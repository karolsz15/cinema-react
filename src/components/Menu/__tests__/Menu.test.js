import React from 'react';
import { shallow } from 'enzyme';
import Menu from '../Menu';

let wrapper = shallow(<Menu />);

describe('Menu component', () => {
  it('should render the Menu correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});