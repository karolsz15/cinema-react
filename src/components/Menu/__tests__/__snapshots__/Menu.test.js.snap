// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should render the Menu correctly 1`] = `
<div
  className="menu"
>
  <ul>
    <li>
      <NavLink
        exact={true}
        to="/"
      >
        <div
          className="home"
        >
          <i
            className="home"
          />
        </div>
      </NavLink>
    </li>
    <li>
      <NavLink
        to="/trailers"
      >
        <div
          className="video"
        >
          <i
            className="videos"
          />
          <i
            className="videos1"
          />
        </div>
      </NavLink>
    </li>
    <li>
      <NavLink
        to="/booking"
      >
        <div
          className="bk"
        >
          <i
            className="booking"
          />
          <i
            className="booking1"
          />
        </div>
      </NavLink>
    </li>
    <li>
      <NavLink
        to="/contact"
      >
        <div
          className="cnt"
        >
          <i
            className="contact"
          />
          <i
            className="contact1"
          />
        </div>
      </NavLink>
    </li>
  </ul>
</div>
`;
